Observer/OrderSaveAfter.php

```diff
@@ -10,9 +10,23 @@
 use Magento\Store\Model\ScopeInterface as SS;
 use Magento\Store\Model\Store;
 use Stock2Shop\OrderExport\Payload;
-use Zend_Http_Client as Z;
 // 2018-08-11 Dmitry Fedyuk https://www.upwork.com/fl/mage2pro
 final class OrderSaveAfter implements ObserverInterface {
+
+	private static $encoding_error_msg = null;
+	private static $curl_error_msg = null;
+
+	protected static $logger;
+
+	/**
+	 * @param \Psr\Log\LoggerInterface $logger
+	 */
+	public function __construct(
+		\Psr\Log\LoggerInterface $logger
```

Comment:
@mannyqi @flumonion Is it reasonable to assume that our customers will have this Psr thing? i.e. is it a basic component of Magento?

Reply:
@matthampton-stock2shop it is the default. Have a look at the docs [here](https://devdocs.magento.com/guides/v2.3/config-guide/log/custom-logger-handler.html)